docs/content/getting-started/setup-openfga.mdx

```diff
@@ -50,10 +50,35 @@ All possible configurations and their default values are defined in [config-sche
 
 ### Configuring Data Storage
 
-You can choose between two storage options:
+OpenFGA supports multiple storage engine options including:
 
-- In-memory storage. This option is not persistent, i.e. if you restart the server, the data will be lost.
-- Postgres database. This option requires either a local Postgres installation, or a Docker container that runs Postgres. For an example of a Docker setup that includes a Postgres database, see [OpenFGA's Docker compose file](https://github.com/openfga/openfga/blob/main/docker-compose.yaml).
+- `memory` - The memory storage engine is the default, but it is not persistent (data is lost between server restarts).
+- `postgres` - The Postgres storage engine requires a Postgres instance that the OpenFGA server can reach.
+
+#### Postgres
+
+Let's assume you want to run OpenFGA and Postgres in containers. The first thing we should do is create a new network as this will make communication between containers a bit simplier:
+```console
+docker network create openfga
+```
+
+Start Postgres in the network you created above:
+```console
+docker run -d --name postgres --network=openfga -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password postgres:14
+```
+
+You should now have Postgres running in a container in the `openfga` network. However, it will not have the tables required for running OpenFGA. You can use the `migrate` command to create the tables. Using the OpenFGA container, this will look like:
+```console
+docker run --rm --network=openfga openfga/openfga migrate --datastore-engine postgres --datastore-uri 'postgres://postgres:password@postgres:5432/postgres?sslmode=disable'
+```
+
+Finally, start OpenFGA:
+```console
+docker run --name openfga --network=openfga -p 3000:3000 -p 8080:8080 -p 8081:8081 openfga/openfga run --datastore-engine postgres --datastore-uri 'postgres://postgres:password@postgres:5432/postgres?sslmode=disable'
+```
+If all went well, OpenFGA should start and you should see `using 'postgres' storage engine` in the logs.
```

Comment:
```suggestion
OpenFGA should start and you should see `using 'postgres' storage engine` in the logs.
```

Sound more confident in the instruction.

Reply:
Thanks!